docs/quality_assurance.md

```diff
@@ -97,41 +98,41 @@ In addition, the implementation MUST guarantee the following [system properties]
+| *Channel Uniqueness* should hold even when a client expires | `TODO` | `??` | `Not planned` | `TODO` | `TODO` |
 
 ---
 
 | Concern re. *Validator Set Replication* | Code Review | Unit Testing | Diff. testing | Testnet | Protocol audit |
 | ------- | ----------- | ------------ | ------------- | ------- | ----- |
-| Every validator set on any consumer chain MUST either be or have been a validator set on the provider chain. | `TODO` | `NA` | `??` | `TODO` | `TODO` |
-| Every consumer chain receives the same sequence of `ValidatorSetChangePacket`s in the same order. | `TODO` | `NA` | `??` | `TODO` | `TODO` |
+| Every validator set on any consumer chain MUST either be or have been a validator set on the provider chain. | `TODO` | `NA` | `Done` | `TODO` | `TODO` |
+| Every consumer chain receives the same sequence of `ValidatorSetChangePacket`s in the same order. | `TODO` | `NA` | `Not planned` | `TODO` | `TODO` |
 
 ---
 
 | Concern re. *Bond-Based Consumer Voting Power* | Code Review | Unit Testing | Diff. testing | Testnet | Protocol audit |
 | ------- | ----------- | ------------ | ------------- | ------- | ----- |
-| A power increase of a validator `val` on a consumer chain can be only due to <br /> - a `Delegate()` / `Redelegate()` to `val` on provider <br /> - `val` joining the provider validator set (another validator leaving the set) | `TODO` | `TODO` | `??` | `TODO` | `TODO` |
-| A power decrease of a validator `val` on a consumer chain can be only due to  <br /> - an `Undelegate()` / `Redelegate()` from `val` on provider <br /> - `val` gets slash on the provider chain <br /> - another validator joining the provider validator set (`val` leaving the set) | `TODO` | `TODO` | `??` | `TODO` | `TODO` |
-| In the case of a power decrease of a validator `val` on a consumer chain at time `t` due to either <br /> - an `Undelegate()` / `Redelegate()` from `val` on provider <br /> - or `val` leaving the set <br /> the corresponding tokens remain locked on the provider chain until at least `t + UnbondingPeriod` | `TODO` | `TODO` | `??` | `TODO` | `TODO` |
+| All power increases of a validator `val` on a consumer chain are because of <br /> - a `Delegate()` / `Redelegate()` to `val` on provider <br /> - `val` joining the provider validator set (another validator leaving the set) | `TODO` | `TODO` | `??` | `TODO` | `TODO` |
+| All power decreases of a validator `val` on a consumer chain are because of <br /> - an `Undelegate()` / `Redelegate()` from `val` on provider <br /> - `val` gets slashed on the provider chain <br /> - another validator joining the provider validator set (`val` leaving the set) | `TODO` | `TODO` | `??` | `TODO` | `TODO` |
+| In the case of a power change of a validator `val` on a consumer chain at time `t` the corresponding tokens remain locked on the provider chain until at least `t + UnbondingPeriod` or are slashed | `TODO` | `TODO` | `Done` | `TODO` | `TODO` |
```

Comment:
If power goes up those tokens should also be locked right?

Reply:
That's implicit since any unlock operation would entail a decrease, which needs to wait for the unbonding period. Anyway, I have change this concern in the main document so it's no longer needed.